core/src/main/java/io/lionweb/lioncore/java/serialization/data/SerializedClassifierInstance.java

```diff
@@ -3,22 +3,23 @@
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
+import java.util.Objects;
 import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
 
 /**
  * Lower level representation of a Classifier Instance (either a Node or an AnnotationInstance)
- * which is used to load broken classifier instances during serialization.
+ * which is used to load classifier instances during serialization.
```

Comment:
I think that the broken term here makes sense as we can use these classes also to load nodes that are not structurally valid (like nodes that have themselves as parent)

Reply:
Yes, but they're not _only_ used for broken classifiers, right? If so, we can state explicitly that this class is also usable to load broken serialized nodes.

Reply:
You are right, I added a suggestion

Reply:
I committed my own suggestion, to get this one over the line